cmd/evm/internal/t8ntool/transition.go

```diff
@@ -152,7 +154,7 @@ func Main(ctx *cli.Context) error {
 		defer inFile.Close()
 		decoder := json.NewDecoder(inFile)
 		if err := decoder.Decode(&inputData.Alloc); err != nil {
-			return NewError(ErrorJson, fmt.Errorf("Failed unmarshaling alloc-file: %v", err))
+			return NewError(ErrorJson, fmt.Errorf("failed unmarshaling alloc-file: %v", err))
```

Comment:
Right, I just recalled why these are upper-case. This is in the `Main`, so these errors not further wrapped, but output on the terminal. Please revert these :)

Reply:
So in this `if`, there is both an upper-case error and a lower case error. If I revert this change, should I also upper case all errors in the `main` function?

Also, I'm not sure if understand what you mean by "further wrapped", it appears the `NewError` wrap the inners error with `ERROR(N): msg`. Are you saying `msg` should be capitalized?

Reply:
This is how it looks now: 
```
 ./build/bin/evm t8n --input.alloc=./asdfadsf
ERROR(11): failed reading alloc file: open ./asdfadsf: no such file or directory
```

I think neither is wrong, and neither is markedly better than the other (for these types of errors)

It's not important really, it could be either way. I just mean that what we typically want to avoid is situations that lead to `Error, could not read data: Failed to open file: Could not baz the foo`.

Reply:
So I guess, let's do the `NewError` with lowercase consistently then, since they're prefixed by `ERROR(xx):` later on

Reply:
Sounds good, I'll update for this.